stable/dex/patches/00-extra-role-and-rolebindings.patch

```diff
@@ -0,0 +1,61 @@
+commit a5158051fc8980586639b500abd1e1bd3352686a
```

Comment:
good idea! I like including the patch files so we can more easily track upstream.